events/eleventh_street.py

```diff
@@ -0,0 +1,169 @@
+
+    return soup
+
+
+
+def schematize_event_time(event_startTime):
+    '''
+    Converts a time string like 'May 30 @ 1:30 pm' to 24hr time like '13:30:00'
+    '''
+
+    try:
+        datetime_obj = datetime.strptime(event_startTime.split(' @ ',1)[1], "%I:%M %p")
+        schematized_event_time = datetime.strftime(datetime_obj, "%H:%M:%S")
+    except IndexError:
+        schematized_event_time = ''
+    except ValueError:
+        logger.warning(f'Exception schematizing this time: {event_startTime}', exc_info = True)
+        schematized_event_time = ''
+
+    return schematized_event_time
+
+def schematize_event_dates(event_id, start_date, end_date):
+    '''
+    Convert a date string like 'May 30 @ 1:30 PM' to a date like 2019-05-30.
+    Checks what year the event is taking place in.
+
+    '''
+    start_date_out = ''
+    end_date_out = ''
+    all_day_event = False
+    try:
+        start_date = datetime.strptime(start_date.split(' @ ', 1)[0], '%B %d')
+        end_date = datetime.strptime(end_date.split(' @ ', 1)[0], '%B %d')
+        id_date = datetime.strptime(event_id[5:], '%Y-%m-%d')
+        start_date = start_date.replace(year = id_date.year)
+        end_date = end_date.replace(year = id_date.year)
+        if start_date > end_date:
+            start_date = start_date.replace(year = id_date.year - 1)
+        if (end_date-start_date).days >= 1:
+            all_day_event = True
+        start_date_out = datetime.strftime(start_date, '%Y-%m-%d')
+        end_date_out = datetime.strftime(end_date, '%Y-%m-%d')
+    except IndexError:
+        pass
+    except ValueError:
+        logger.warning(f'Exception schematizing this date: {start_date}', exc_info = True)
+        logger.warning(f'Exception schematizing this date: {end_date}', exc_info = True)
+    return start_date_out, end_date_out, all_day_event
+
+def get_event_description(event_website_soup, event_name):
+    description = ''
+    try:
+        soup_paragraphs = event_website_soup.find('div', 'tribe-events-single-event-description').find_all('p')
+        soup_paragraphs = [i.text for i in soup_paragraphs]
+        description = ''.join(soup_paragraphs)
+    except:
+        logger.warning(f'Exception finding this event description: {event_name}', exc_info = True)
+    return description
+
+def get_event_venue(event_website_soup, event_name):
+    venue = ''
+    try:
+        venue = event_website_soup.find('dd', class_ = 'tribe-venue').text
+    except:
+        logger.warning(f'Exception finding this event venue: {event_name}', exc_info = True)
+    return venue.strip()
+
+def get_event_category(event_website_soup, event_name):
+    category = ''
+    try:
+        category = event_website_soup.find('dd', class_ = 'tribe-events-event-categories').text
+    except AttributeError:
+        #print(event_name)
+        category = ''
+    except:
+        logger.warning(f'Exception finding this event category: {event_name}', exc_info = True)
```

Comment:
Could you edit this log message so that it says `f'Exception finding the event category for {event_name}'`